proxy/src/main.rs

```diff
@@ -75,6 +75,13 @@ fn main() -> Result<(), Box<dyn Error>> {
                 .default_value("6379")
                 .value_name("PORT")
                 .help("Specify the redis-port to connect to"),
+        ).arg(
+            Arg::with_name("sentinel")
+                .long("sentinel")
```

Comment:
```suggestion
                .short("m")
                .long("sentinel")
```

Add short to match worker.

Reply:
Solved in 5ef873ea7